validator/submain.py

```diff
@@ -386,6 +386,7 @@ def test_inner_package(err, xpi_package, for_appversions=None):
     if (err.get_resource('has_install_rdf') and
             not err.get_resource('has_manifest_json') and
             not err.get_resource('is_multiprocess_compatible') and
+            err.detected_type == PACKAGE_EXTENSION and
```

Comment:
though, explicit is better than implicit regarding the inclusion of both `PACKAGE_MULTI` and `PACKAGE_EXTENSION` so I'm actually curious why we have separate constants if we don't use them :-/ But I gues that's "legacy".

No idea what exactly a subpackage is though :disappointed: